releases/EXCEPTIONS.md

```diff
@@ -44,6 +44,9 @@ Email them to:
 
 [You should have *very high confidence* on the “additional time needed” number - we will not grant multiple exceptions for a enhancement. The same goes for over-estimating on the time needed. If the exception request is rejected on the basis that the asking time is too high, we will not re-evaluate if the asking time decreases.]
 
+Exceptions after PRR freeze:
+Due to the proximity to enhancements freeze, it is important to file an exception at your earliest convenience. You may file an exception request in advance if you know your KEP will miss the PRR freeze deadline. All requests will be reviewed and either approved or rejected as they come in.  The release team will be responsible for approving or rejecting exceptions based on the questions above.  
```

Comment:
Should we define a hard deadline to submit PRR exception requests (e.g. X days after PRR freeze), and specify when the updates need to be done by? I'm afraid that if they come too close to enhancements freeze it will be like a domino effect and they'll have to submit an exception request for enhancements freeze too. Can one have both a PRR freeze and enhancements freeze exception request open at the same time?